src/dbnode/storage/block/block.go

```diff
@@ -354,10 +363,10 @@ func (b *dbBlock) streamWithRLock(ctx context.Context) (xio.BlockReader, error)
 // TODO(rartoul): The existing ctx is still holding a reference to the old segment so that will hang around
 // and waste memory until the block is closed. We could improve this by swapping out the underlying ctx with
 // a new one, allowing us to close the old one and release the old segment, freeing memory.
-func (b *dbBlock) forceMergeWithLock(ctx context.Context, stream xio.SegmentReader) (xio.BlockReader, error) {
+func (b *dbBlock) forceMergeWithLock(ctx context.Context, stream xio.SegmentReader) error {
```

Comment:
you can delete the comment above this yea?

Reply:
woosp yeah meant to do that